features/lorawan/system/LoRaWANTimer.h

```diff
@@ -0,0 +1,115 @@
+/**
+ / _____)             _              | |
+( (____  _____ ____ _| |_ _____  ____| |__
+ \____ \| ___ |    (_   _) ___ |/ ___)  _ \
+ _____) ) ____| | | || |_| ____( (___| | | |
+(______/|_____)_|_|_| \__)_____)\____)_| |_|
+    (C)2013 Semtech
+
+Description: Timer objects and scheduling management
+
+License: Revised BSD License, see LICENSE.TXT file include in the project
+
+Maintainer: Miguel Luis and Gregory Cristian
```

Comment:
Is this correct? we got git for this, I would remove these type of lines from our files.

Reply:
We can't. The code comes from Semtech, and we need to respect their BSD license.